ValetWatchTest Extension/InterfaceController.m

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
This class is a good start (thanks for following `ValetTouchIDTest`). But we're going to need significantly more tests before we can merge this. I was hoping to have a button on the watch run all unit tests – this is likely a significant undertaking. Right now I can use this PR to test SecureEnclave, but I don't know if any other code is broken. Being abel to run all our unit tests would help with this.

Please take a look at the full checklist from https://github.com/square/Valet/issues/40#issuecomment-138634007